webui/react/src/utils/error.ts

```diff
@@ -76,6 +79,7 @@ export class DetError extends Error {
     this.publicSubject = options.publicSubject || detError?.publicSubject || undefined;
     this.silent = options.silent || detError?.silent || false;
     this.type = options.type || detError?.type || ErrorType.Unknown;
+    this.isHandled = false;
```

Comment:
the high-level idea is that each error needs to be handled only once and it should happen closest to where it has the most information on the cause.

eg (this was happening prior to this)

low in the callstack, eg processApiError, the error is handled and rethrown. I'm thinking probably a usecase we want to avoid. either handle or just create/augment a DetError and throw.
then at a higher level, eg where use had clicked forkExperiment button. they want to provide an error notification and break the flow of a successful fork then if they use handleerror it would do a double notification. 

this helps avoid getting into the double handling situation but it might be better if it didnt' silently ignore it 🤔 

also, another scenario is if we have a catch-all wrapper. it can know whether a DetError should be ignored or needs to be handled. it feels like it has some flaws